.github/workflows/ci.yaml

```
@@ -13,6 +13,10 @@ on:
     - track/**
 
 jobs:
+  build:
```

maybe more explicit ?
```suggestion
  build-charms:
```